generator/postgres/query_set.go

```diff
@@ -44,6 +44,7 @@ WITH primaryKeys AS (
 )
 SELECT column_name as "column.Name", 
 	   is_nullable = 'YES' as "column.isNullable",
+	   is_generated = 'ALWAYS' as "column.isGenerated",
```

Comment:
We will need to update this condition since `cockroachdb` doesn't use `ALWAYS` but `YES` as a constant.

Reply:
Do you have an example in the code where we send one way for PostgreSQL and another way for CockroachDB?

Reply:
This query is used for both databases. Cockroach shares interfaces with postgres, but there are occasionally some differences. So we just have to add: `is_generated = 'ALWAYS' or is_generated = 'YES'`

Reply:
Seems ok in local tests.

Reply:
Yep, looks good.